src/_includes/layouts/homepage.html

```diff
@@ -26,7 +26,8 @@ <h2 class="events-title">Upcoming Events</h2>
 					<h3 id="{{ event.data.title | slug }}">{{ event.data.title }}</h3>
 
 					<p class="fg1 block subtitle">
-						{% if event.data.type %}{{ event.data.type }} &middot; {% endif %} {{ event.data.date || asDateTime }}
+						<span>{% if event.data.type %}{{ event.data.type }} &middot; {% endif %}</span>
+						<time data-datetime datetime="{{event.data.date | toISOString }}">{{ event.data.date || asDateTime }}</time>
```

Comment:
Hmmm... the orphaned `data-datetime` attribute looks a little weird to me now, as it's not immediately clear to me what it's for.

I think I'd iterate on this in one of two different ways.

1. Lean into how this is basically a boolean attribute now, and name it `data-localizedatetime` or something different.
2. Remove it, and use `[datetime]` as our selector when deciding what to localize. This means people can use semantics and the localization "just works."

I'm hesitant to do the second one, however, because it means *all* `<time datetime>` elements will get client-side localized, no matter what - no way to opt in or out.

Reply:
^I also want to acknowledge that this site is so small that I think my wavering on the second option here is completely academic. The chances of us having more `<time>`s we _don't_ want localized are practically very slim.

Reply:
good catch!

yes. i'd left it to avoid the possibly incorrect long-term assumption of #2.

#1 (reaming) is a very good option. here are some additional options:

1. `data-localized-datetime`
1. `data-js-replace-content="datetime"`
1. `data-datetime="localize"`
1. `data-datetime="localize-and-replace-content"`

I actually really like #2. seems the most obvious to me, just seeing the markup

Reply:
option #2 illustrated in this commit: https://github.com/LunchDevCommunity/community-calendar/pull/101/commits/96e55113c359c166116b5b4418897508b8990717

Reply:
I'm genuinely struggling to see a case when we might _not_ want to have a localizable `<time>` on this particular site, so if you want to go Option 2, I'd support that.

Your alternate names for Option 1 are good. I think if I were to pick one of your names, I'd go with `data-localized-datetime`.

Reply:
cool. i'm honestly down to roll with `<time datetime>` and seeing where it goes :)
i'll change it to that